### PR TITLE
exerciseHandlerspec and exercise-testing-util

### PR DESCRIPTION
**Pull request by zacharyasis**
updated generateRandomExercise() parameters
added exerciseHandlerspec

### CODE REVIEW

**Comment by zacharyasis** [comment]
I'm not sure why it says so many files were changed it's just the exercise handler specs/util?

File: spec/support/utils/exercise-testing-util.js [hunk 1]
```diff
@@ -7,13 +7,8 @@ function generateRandomRestInterval() {
 }
 
 
-const generateRandomExercise = (userId, exerciseId) => {
-    let name = generateRandomString(6);
-    let force = generateRandomEnumValue(enums.enumList.forceEnum);
```
Comment (PietropaoloV) [inline spec/support/utils/exercise-testing-util.js:12]: You should leave the random generation and maybe make another method that overloads this with the same name. I understand the need to have null fields, but making an overloaded method is the best (a separate method)

**Review by PietropaoloV — CHANGES_REQUESTED** [review]

File: spec/support/utils/exercise-testing-util.js (hunk 1 above)
Comment (zacharyasis) [reply spec/support/utils/exercise-testing-util.js:12]: I'm not sure the best way to implement this since Javascript doesn't support overloading

Comment (PietropaoloV) [reply spec/support/utils/exercise-testing-util.js:12]: no but it supports default parameters if a parameter isnt preset make it auto generated

**Comment by PietropaoloV** [comment]
> I'm not sure why it says so many files were changed it's just the exercise handler specs/util? How do I fix that?

remove the nodemodules folder make a commit and push that

**Pull request closed without merging**